### PR TITLE
feat: implement help menu

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: src/menus/UserInfo/UserInfo.js [hunk 1]
```diff
@@ -59,6 +56,11 @@ export const UserInfo = (props) => {
   const [anchorEl, setAnchorEl] = useState(null);
   const [langAnchorEl, setLangAnchorEl] = useState(null);
 
+  const handleClose = () => {
+    setIsMenuOpened(false);
+    setLangIsMenuOpened(false);
```
Comment (COSMO-VMI) [inline src/menus/UserInfo/UserInfo.js:61]: Please name them `isLanguageMenuOpen` and `setLanguageMenuOpen` instead.

File: src/menus/HelpMenu/HelpMenu.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    fontSize: 32,
+  },
+  link: {
+    color: theme.palette.primary.contrastText,
+  },
+}));
+
+export const HelpMenu = (props) => {
+  const classes = useStyles();
+  const [isMenuOpened, setIsMenuOpened] = useState(false);
```
Comment (COSMO-VMI) [inline src/menus/HelpMenu/HelpMenu.js:43]: Please name them `isMenuOpen` and `setMenuOpen` instead.

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]
Just some setters and getters to rename, LGTM.

File: src/menus/UserInfo/UserInfo.js [hunk 3]
```diff
@@ -70,91 +72,81 @@ export const UserInfo = (props) => {
...
+          aria-haspopup="true"
+          onClick={handleClick}
+          className={`${classes.menuTrigger} ${isMenuOpened ? 'active' : ''}`}
+        >
+          <img className={classes.profilePic} src={profilePlaceholder} />
+          <span className={classes.userName}>{userName}</span>
+        </Box>
+
         <Menu
           className={classes.menu}
```
Comment (jreynard-code) [inline src/menus/UserInfo/UserInfo.js:96]: could you please add an `id` here?

File: src/menus/UserInfo/UserInfo.js [hunk 4]
```diff
@@ -70,91 +72,81 @@ export const UserInfo = (props) => {
...
+          <MenuItem data-cy="logout" onClick={onLogout}>
+            {labels.logOut}
+          </MenuItem>
         </Menu>
-      )}
-    </React.Fragment>
+        {languages && (
+          <Menu
+            className={classes.menu}
+            id="simple-menu"
```
Comment (jreynard-code) [inline src/menus/UserInfo/UserInfo.js:116]: Could you please change the `id` value? it'd be better named 'languages-menu

**Pull request merged**